pennylane/ops/qubit/matrix_ops.py

```diff
@@ -73,15 +74,15 @@ def __init__(self, *params, wires, do_queue=True):
 
             dim = 2 ** len(wires)
 
-            if not (len(U_shape) in {2, 3} and U_shape[-2:] == (dim, dim)):
+            if len(U_shape) not in {2, 3} or U_shape[-2:] != (dim, dim):
                 raise ValueError(
                     f"Input unitary must be of shape {(dim, dim)} or (batch_size, {dim}, {dim}) "
-                    f"to act on {len(wires)} wires."
+                    + "to act on {len(wires)} wires."
```

Comment:
I don't think you need the plus here? Anyways just a suggestion. 
```suggestion
                      "to act on {len(wires)} wires."
```